starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,53 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
-
-// Chronometer.prototype.setMilliseconds = function () {
-
-// };
-
-// Chronometer.prototype.stopClick = function () {
-  
-// };
-
-// Chronometer.prototype.resetClick = function () {
-
-// };
-
-// Chronometer.prototype.splitClick = function () {
-
-// };
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId = 0;
+}
+
+Chronometer.prototype.startClick = function () {
+  var self = this;
+  var callbackFunction = function (){
```

Comment:
next step: declaring the callback inside the setInterval. It will be something like:
```
this.intervalId = setInterval(function() {
     self.currentTime += 1;
     self.setTime();
}, 1000);
```